packages/web-components/.storybook/preview-head.html

```diff
@@ -1,3 +1,5 @@
+<link rel="stylesheet"
+  href="https://static2.sharepointonline.com/files/fabric/office-ui-fabric-core/11.0.0/css/fabric.min.css" />
```

Comment:
While I think this made sense to test with for your scenario, I'd like to avoid adding this globally for now to avoid confusion with future plans and the current POR guidance of leveraging the Fluent icons. If you'd like to open an issue to discuss a more in-depth solution for that, we'd welcome it!